queue_services/entity-emailer/src/entity_emailer/email_processors/bn_notification.py

```diff
@@ -12,13 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Email processing rules and actions for business number notification."""
+from __future__ import annotations
+
 from entity_queue_common.service_utils import logger
 
 
-def process(email_msg):
+def process(email_msg: dict) -> dict:
     """Build the email for Business Number notification."""
     logger.debug('bn_notification: %s', email_msg)
-    email = 'bn hardcoded'
     # build email from template
     # get links etc.
-    return email
+    return {
+        'recipients': 'hardcoded@email.com',
+        'requestBy': '',
+        'content': {
+            'subject': 'hardcoded',
+            'body': 'hardcoded',
```

Comment:
Should these have correct values before merging into master?

Reply:
No - they aren't implemented yet

Reply:
it's just a placeholder for the dev who's in there next

Reply:
I'd be tempted to use the word "future" in a comment here, but it's up to you.